SoilMoistureSensor/SoilMoistureStatus.cs

```diff
@@ -0,0 +1,23 @@
+﻿//Console.WriteLine();
+
+//var client = DeviceClient.CreateFromConnectionString("HostName=ECM3440.azure-devices.net;DeviceId=soil-moisture-sensor;SharedAccessKey=VGlo7rYmMZJVcVaJIehiOgF0QnUCmh3FHFxfpu9RukY=");
+
+// IOT connection
+// Don't need CounterFit - can just send any data
+
+
+// Interface to provide generic constraints
+// Message class for temperature values
+using SoilMoistureSensor;
+
+public class SoilMoistureStatus : IIoTHubMessage
+{
+	public double Value { get; set; }
```

Comment:
If I see one more public set I'm going to write to Kent Beck and ask him to take me home